docs/notebooks/mesh_to_mesh.ipynb

```diff
@@ -34,7 +34,7 @@
       "outputs": [],
       "source": [
         "!pip install 'finetuner[full]'\n",
-        "!pip install 'docarray[full]'"
+        "!pip install 'docarray[full]==0.21.0'"
```

Comment:
```suggestion
        "!pip install 'docarray[full]<0.3.0'"
```

just a suggestion. It might happen that we need to do a fix on a earlier version of docarray. All o the version between 0.2 and 0.3 and for legacy docarray